mmseg/models/losses/cross_entropy_loss.py

```diff
@@ -161,14 +204,16 @@ def __init__(self,
                  reduction='mean',
                  class_weight=None,
                  loss_weight=1.0,
-                 loss_name='loss_ce'):
+                 loss_name='loss_ce',
+                 avg_non_ignore=False):
```

Comment:
might add a warning when `avg_non_ignore` is False to tell users if they would like to ignore some pixels they must open `avg_non_ignore` manually, `avg_non_ignore` defaults to closed